frontend/src/libs/components/input/input.tsx

```diff
@@ -11,30 +11,37 @@ import styles from './styles.module.scss';
 
 type Properties<T extends FormFieldValues> = {
   control: FormControl<T, null>;
-  errors: FormFieldErrors<T>;
+  errors?: FormFieldErrors<T>;
```

Comment:
```suggestion
  errors: FormFieldErrors<T>;
```
It shouldn't be optional

Reply:
solved